briana_bennett/python_fundamentals/Coin Tosses.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+def coin_tosses():
+    print "Starting the program..."
+    heads_count = 0
+    tails_count = 0
+    coin_val = ""
+
+    for flips in range(1, 51):
+        rand_int = random.randint(0, 1)
+        if rand_int is 0:
+            coin_val = "head"
+            heads_count += 1
+        if rand_int is 1:
+            coin_val = "tail"
+            tails_count += 1
+
+        print "Attempt #" + str(flips) + ": Throwing a coin... It's a " + str(coin_val) + "! ... Got " + str(heads_count) + " head(s) so far and " + str(tails_count) + " tail(s) so far."
+

```

Comment:
the way you printed your string is fine.  although, i would consider trying the `.format()` method, because you won't have to worry about converting your variables to strings (such as `str(flips)`).  It would look like this...
``` Python
print "Attempt #{}: Throwing a coin... It's a {}! ... Got {} head(s) so far and {} tail(s) so far.".format(flips, coin_val, heads_count, tails_count)
```